submissions/emmanuelEsuong/javaScript/signUp.js

```diff
@@ -0,0 +1,177 @@
+let submitForm = document.getElementById('validateForm'),
+    firstName = document.getElementById('firstName'),
+    lastName = document.getElementById('lastName'),
+    email = document.getElementById('email'),
+    number = document.getElementById('number'),
+    passWord = document.getElementById('password'),
+    confirmPassword = document.getElementById('confirmPassword'),
+    terms = document.getElementById('terms')
+    ;
+
+    //required fields function
+     function required(field, event) {
+        if (field.value === '') {
+            field.nextElementSibling.innerHTML = "This field is required";
+            event.preventDefault();
+            return false;
+        }   else {
+            field.nextElementSibling.innerHTML = "";
+            return true;
+        }
+    }
+
+    //name fields validation regex
+    function lettersOnly(firstName, lastName, otherName) {
```

Comment:
Well done @ElJentle But it's like you are using too many energy to do a little job

This function, I don't think is necessary.